src/searching/searching.py

```diff
@@ -2,8 +2,17 @@
 def linear_search(arr, target):
   
   # TO-DO: add missing code
-
-   return -1   # not found
+  arr = [1, 3, 50, 15, 23, 11, 8, 2, 118]
+  x = 10
+  len = len(arr)
+  
```

Comment:
Good good! Linear search is like the worst kind of search, hahahah, but it's still like the go-to when you need it.